Makefile.dep

```diff
@@ -573,3 +573,8 @@ USEPKG := $(sort $(USEPKG))
 ifneq ($(OLD_USEMODULE) $(OLD_USEPKG),$(USEMODULE) $(USEPKG))
     include $(RIOTBASE)/Makefile.dep
 endif
+
+USEPKG_BUILDFIRST := nordic_softdevice_ble
```

Comment:
Doesn't this ordering problem persist for packages in `USEPKG_BUILDFIRST` with the proposed solution?